components/icon/Flag.vue

```diff
@@ -1,22 +1,22 @@
-<style>
-
-</style>
 <template>
-    <span v-if="iso" class="flag-icon" :class="flagIconClass" :title="title || iso">
-    </span>
+  <span v-if="iso" class="fi" :class="flagIconClass" :title="title || iso">
+  </span>
 </template>
+
 <script>
 export default {
-    name: 'flag',
-    props: {
-        iso: { type: String, default: null },
-        title: { type: String, default: null },
-        squared: { type: Boolean, default: true },
+  name: "flag",
+  props: {
+    iso: { type: String, default: null },
+    title: { type: String, default: null },
+    squared: { type: Boolean, default: true },
+  },
+  computed: {
+    flagIconClass() {
+      return (!!this.squared ? "fis " : "") + "fi-" + this.iso.toLowerCase();
```

Comment:
Could also be changed to
 return  `${this.squared ? "fis" : ""}  fi-${this.iso.toLowerCase()}`;

Reply:
good shout @EdgarKisman, what do you think @rafalolszewski94?

Reply:
will amend this and add the readme changes of #23 into a new branch, then look up at the vue3 support